portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -726,13 +726,59 @@ definition {
 	}
 
 	@description = "Verify that a Form can be unpublished"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeUnpublished {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-130507 CanBeUnpublished pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Paragraph");
```

Comment:
Hi @LucasAcacio ,

I saw that you are using the Paragraph field to create the form but reading the test specification it's not mandatory to use the Paragraph field. 

The Paragraph field is a little heavy to render than other simple fields like (Text, Numeric ... ) and also it requires an extra step to add the paragraph content(this could cost precious seconds on test execution time). So if you need only to use a simple field to be able to create a form I would suggest you use the Text or Numeric field.

Thanks!